test/evergreen.yml

```diff
@@ -2407,6 +2407,151 @@ tasks:
             cp ../../../bench/wtperf/split_heavy.wtperf .
             ./wtperf -O ./split_heavy.wtperf -o verbose=2
 
+  - name: data-validation-stress-test-checkpoint
+    depends_on:
+    - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix/test/checkpoint"
+          shell: bash
+          script: |
+            set -o errexit
+            set -o verbose
+
+            CMD='nice ../../../test/checkpoint/recovery-test.sh "-t r -W 3 -D -n 100000 -k 100000 -C cache_size=100MB" WT_TEST.$t'
+
+            NPROCS=$(echo "`grep -c ^processor /proc/cpuinfo` / 2" | bc)
+
+            outf=./outfile.txt
+            for i in $(seq 120); do
+              echo "$d: Iteration $i" >> $outf
+              echo "$d: Iteration $i"
+              for((t=0; t<$NPROCS; t++)); do
+                > nohup.out.$t ; eval nohup $CMD > nohup.out.$t 2>&1 &
+              done
+
+              for((t=0; t<$NPROCS; t++)); do
+                wait -n || exit $?
+              done
+            done
+
+  - name: data-validation-stress-test-checkpoint-fp-hs-insert-1
+    depends_on:
+    - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix/test/checkpoint"
+          shell: bash
+          script: |
+            set -o errexit
+            set -o verbose
+
+            CMD='nice ../../../test/checkpoint/recovery-test.sh "-t r -W 3 -s 5 -D -n 100000 -k 100000 -C cache_size=100MB" WT_TEST.$t'
+
+            NPROCS=$(echo "`grep -c ^processor /proc/cpuinfo` / 2" | bc)
+
+            outf=./outfile.txt
+            for i in $(seq 120); do
+              echo "$d: Iteration $i" >> $outf
+              echo "$d: Iteration $i"
+              for((t=0; t<$NPROCS; t++)); do
+                > nohup.out.$t ; eval nohup $CMD > nohup.out.$t 2>&1 &
+              done
+
+              for((t=0; t<$NPROCS; t++)); do
+                wait -n || exit $?
+              done
+            done
+
+  - name: data-validation-stress-test-checkpoint-fp-hs-insert-s2
+    depends_on:
+    - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix/test/checkpoint"
+          shell: bash
+          script: |
+            set -o errexit
+            set -o verbose
+            ../../../tools/run_parallel.sh 'nice ../../../test/checkpoint/recovery-test.sh "-t r -W 3 -s 2 -D -n 100000 -k 100000 -C cache_size=100MB" WT_TEST.$t' 120
```

Comment:
It looks all newly added tasks have some common format config setttings, i.e. `-W 3 -D -n 100000 -k 100000 -C cache_size=100MB`. Can we turn those common parts into a CONFIG file (e.g. test/format/CONFIG.checkpoint) to make the code neater?

Reply:
I just realized it's test/checkpoint not test/format (though both compile into a `t` program). Please disregard this comment.